README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 01
+
+#**Sarah Jensen**
+##Intro
+###My name is Sarah, and I am a 2018 graduate of St. Norbert College where I earned my B.A. in Psychology and French. I study moral psychology and am interested in how children develop their sense of right and wrong based on their social environments. 
+
```

Comment:
That's an extremely interesting field of study! I'd love to know more about how morality arises and what social factors influence its evolution.